api/apps/services/dataset_api_service.py

```
@@ -3348,6 +3348,434 @@ async def search_rows(condition):
+        ranks = await settings.retriever.retrieval(
+            query,
+            embd_mdl,
+            tenant_ids,
+            [dataset_id],
+            1,
+            fetch_k,
+            0.0,
+            0.3,
+            **kwargs,
+        )
+    except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)